torch/_inductor/decomposition.py

```diff
@@ -521,7 +521,7 @@ def dequantize_per_tensor_tensor_decomp_impl(
     quant_max: int,
     dtype: torch.dtype,
 ) -> torch.Tensor:
-    return (input.to(torch.float32) - zero_point) * scale
+    return (input.to(torch.float32) - zero_point) * scale.to(torch.float32)
```

Comment:
nit: maybe change `zero_point` to `zero_point.to(torch.int32)` as well

Reply:
Thanks. It's changed.